responses/en/HassGetWeather.yaml

```diff
@@ -0,0 +1,7 @@
+language: en
+responses:
+  intents:
+    HassGetWeather:
+      default: "It is now {{ state.attributes.temperature }} and {{ state.state }}"
+
+      forecast: "{{ slots.timeframe | capitalize }} it will be {{ state.attributes.forecast.condition }} with a high of {{ state.attributes.forecast.temperature }} and a low of {{ state.attributes.forecast.templow }}"
```

Comment:
This is not going to work. I think that for each state of the weather entity we need to get a description going. 

Here are the different states that we need a description for: https://developers.home-assistant.io/docs/core/entity/weather#recommended-values-for-state-and-condition

Reply:
For locks, binary sensors et. al. we are getting a translated state string from the core. Why does this need to be different?

Reply:
I don't think that we can get a translation that makes a grammatically perfect sentence for each condition. It would be nicer if we can have perfect sentences.

Condition | Description
-- | --
clear-night | Clear night
cloudy | Many clouds
exceptional | Exceptional
fog | Fog
hail | Hail
lightning | Lightning/ thunderstorms
lightning-rainy | Lightning/ thunderstorms and rain
partlycloudy | A few clouds
pouring | Pouring rain
rainy | Rain
snowy | Snow
snowy-rainy | Snow and Rain
sunny | Sunshine
windy | Wind
windy-variant | Wind and clouds